conf/messages.en

```diff
@@ -196,8 +196,8 @@ individualBeneficiaryIncomeYesNo.hint = See the trust deed for details.
 individualBeneficiaryIncomeYesNo.error.required = Select yes if the trustees have discretion over how much income the individual beneficiary may get
 
 individualBeneficiaryIncome.title = What percentage of the trust’s assets is the individual beneficiary entitled to?
```

Comment:
We should be changing the content for the titles too. Please check, seems to be a lot of occurrence missed.